clients/ops/privacy-center/package.json

```diff
@@ -17,22 +17,17 @@
     "build-privacy-portal": "npm run build"
   },
   "dependencies": {
-<<<<<<< HEAD
-    "@chakra-ui/react": "^1.7.4",
-    "@chakra-ui/system": ">=1.0.0",
-=======
     "@chakra-ui/icons": "^1.1.7",
     "@chakra-ui/react": "^1.7.4",
     "@chakra-ui/system": "^1.12.1",
->>>>>>> unified-fides-2
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
     "@fidesui/react": "^0.0.9",
     "@fontsource/inter": "^4.5.4",
     "formik": "^2.2.9",
     "framer-motion": "^5",
     "headers-polyfill": "^3.0.10",
-    "next": "12.1.0",
+    "next": "12.2.5",
```

Comment:
I wasn't able to do an `npm i` without upgrading next since the `next-auth` version getting installed required a more recent version of next.